app/main/lib/elastic_crud.py

```diff
@@ -54,7 +54,7 @@ def get_blocked_presto_response(task, model, modality):
     callback_url = Presto.add_item_callback_url(app.config['ALEGRE_HOST'], modality)
     if requires_encoding(obj):
         blocked_results = []
-        for model_key in obj.pop("models", []):
+        for model_key in obj.get("models", []):
```

Comment:
Is the problem that we are destructively modifying the 'obj' object when the "models" list is removed?  It seems like it shouldn't make a difference since it isn't referenced later, but .get seems like better syntax.

Reply:
when we send these requests out, with the pop in place, it "tells" the alegre callback that is expecting fewer models than it really should be - by enforcing the get we (a) bring it in line with async and (b) probably eliminate erroneous calls back to check-api when alegre mistakenly things work is complete for every in-process vector